rbac/api/cross_access/model.py

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Models to store cross account access request."""
+import datetime
+from uuid import uuid4
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils import timezone
+from management.rbac_fields import AutoDateTimeField
+
+
+STATUS_LIST = ["pending", "cancelled", "approved", "denied", "expired"]
+
+
+class Employee(models.Model):
```

Comment:
@lphiri any concerns with storing PII for employees on these requests?

Reply:
@coderbydesign checking - email sent out.